setup.py

```diff
@@ -49,8 +49,9 @@ def readme():
                       'plams@git+https://github.com/SCM-NV/PLAMS@master',
                       'pyparsing', 'pyyaml>=5.1', 'filelock'],
     extras_require={
-        'test': ['pytest', 'pytest-cov', 'pytest-mock', 'typing_extensions',
-                 'assertionlib', 'pycodestyle', 'mypy'],
+        'test': ['assertionlib', 'mypy', 'pycodestyle', 'pytest', 'pytest-cov',
```

Comment:
Do we still need ``'pycodestyle'`` here if we have ``'pytest-pycodestyle'``?

Reply:
not, because the pytest extension will install it. I am going to delete :)